web/api/swagger/mocks.yaml

```diff
@@ -723,24 +809,87 @@ definitions:
         type: "string"
         format: "date-time"
         description: "Timestamp job started"
-      output:
+      steps:
         type: "array"
-        description: "Array with Job output info (not required for UI atm - skip implementation)"
+        description: "Array of Step output info"
         items:
-          type: "string"
-      stepId:
```

Comment:
From: https://wiki.marklogic.com/display/ENGINEERING/DHF+Web+Service+REST+API+Data+Model?showComments=true#DHFWebServiceRESTAPIDataModel-Ex.Step:Mastering
Job should contain:

    "stepId": "my-ingest-step-1-ingest",
    "stepName": "My Ingest Step 1",
    "stepRunningPercent": null,

The addition of "lastAttemptedStep" and "lastCompleted" step don't match this structure.  By default, the "stepName" & "stepId" reflect the step being currently executed or, in the case of a failed job, the last step that executed.  So all we need are "stepName" & "stepId".